generateBins.py

```diff
@@ -22,6 +30,48 @@
 # Following bins are for Rwanda
 POPULATION_BINS = [2125, 5640, 8989, 12108, 15577, 20289, 27629, 49378, 95262, 286928]
 
+# getting data ready for binning
+# data should be 1-dimensional array, python list or iterable
+myArray  = np.loadtxt("GIS\\rwa_population.asc", skiprows = 6)
+print (type(myArray))
+print(myArray.ndim)
+print(myArray.shape)
+print(myArray.size)
+array_1d = myArray.flatten()
+print (array_1d)
+print(array_1d.ndim)
+print(array_1d.shape)
+print(array_1d.size)
+#gvf = 0.0
+#nclasses = 20
+
+def goodness_of_variance_fit(array, classes):
```

Comment:
This should be moved over to the calibrationLib.py since it's quite likely it will end up getting used by something else in the future.